app/views/account/admin/companies/index.html.slim

```diff
@@ -0,0 +1,9 @@
+.container-wrapper
+  .new-button
+    = link_to new_account_admin_company_path, class: 'btn btn-primary'
+      = t('companies.index.btn_new_company')
+  h1 #{t('companies.index.companies_title')}
+  =render 'search_form'
+  .row.mt-4
+    - @companies.each do |company|
+      = render company
```

Comment:
please add pagination in this PR, or later, don't forget about it. Look at users page